test/gtest/common/test.cc

```diff
@@ -248,9 +230,15 @@ void test_base::TearDownProxy() {
         cleanup();
     }
 
-    ucs_log_pop_handler();
     m_errors.clear();
 
+    ucs_log_pop_handler();
```

Comment:
should we use RAII here too?

Reply:
I don't think that calling destructor implicitly is a RAII... more classes in inheritance to define ordering of destructors will only complicate the code. Do you know how to use it other way?